src/orient/rational-calc.orient

```diff
@@ -0,0 +1,614 @@
+  describe(rig_storage_min, "Minimum storage required to mine", bytes)
+  describe(rig_storage_min_tib, "Minimum storage required to mine", tib)
+  describe(rig_storage_parallelization_min, "Minimum storage units that must be accessed in parallel in order to mine (honest)")
+  describe(rig_storage_unit_max, "Maximum storage capacity for a storage unit", bytes)
+  describe(rig_storage_unit_max_tib, "Maximum storage capacity for a storage unit", tib)
+
+  describe(rig_malicious_cost_per_year, "Cost of the malicious computing rig per year")
+  describe(rig_cores, "Number of CPU cores")
+  describe(rig_snark_parallelization, "Number of SNARK proofs that can be generated in parallel")
+  describe(rig_storage_read_mbs, "Sequential read speed (MB/s)")
+  describe(rig_storage_read_mibs, "Sequential read speed (MiB/s)")
+
+  rig_storage_read_mibs = rig_storage_read_mbs * to_i
+  rig_storage_min = rig_storage_min_tib * b_in_tib
+  rig_sectors_min = rig_storage_min / sector_size
+  rig_storage_parallelization_min = epost_data_access / polling_time
+
+  rig_storage_unit_max = rig_storage_min / rig_storage_parallelization_min
+  rig_storage_unit_max_tib = rig_storage_unit_max / b_in_tib 
+
+Crypto:
+  describe(snark_size, "Size of a SNARK proof", bytes)
+  describe(snark_constraint_time, "Time of proving a SNARK constraint", secs)
+  describe(hashing_amax, "Factor of difference between stock CPU hashing and the fastest hashing")
+  describe(merkle_tree_datahash_constraints, "Constraints for hashing two nodes in CommD")
+  describe(merkle_tree_datahash_time, "Time to hash two nodes in CommD", secs)
+  describe(merkle_tree_hash_constraints, "Constraints for hashing two nodes in CommR")
+  describe(merkle_tree_hash_time, "Time to hash two nodes in CommR", secs)
+  describe(column_leaf_hash_constraints, "Constraints for hashing a single row in a column", secs)
+  describe(column_leaf_hash_time, "Time to compute the hash of a single row in a column", secs)
+
+Utils:
+  describe(cid_size, "Size of a CID", bytes)
+  describe(secs_in_month, "Number of seconds in a month")
+  describe(secs_in_year, "Number of seconds in a year")
+  describe(secs_in_day, "Number of seconds in a day")
+  describe(b_in_eib, "Number of bytes in a EiB")
+  describe(b_in_pib, "Number of bytes in a PiB")
+  describe(b_in_tib, "Number of bytes in a TiB")
+  describe(b_in_gib, "Number of bytes in a GiB")
+  describe(b_in_mib, "Number of bytes in a MiB")
+  describe(b_in_kib, "Number of bytes in a KiB")
+  describe(to_i, "Ratio between GB and GiB")
+
+  b_in_eib = (1024 * 1024 * 1024 * 1024 * 1024 * 1024)
+  b_in_pib = (1024 * 1024 * 1024 * 1024 * 1024)
+  b_in_tib = (1024 * 1024 * 1024 * 1024)
+  b_in_gib = (1024 * 1024 * 1024)
+  b_in_mib = (1024 * 1024)
+  b_in_kib = 1024
+  secs_in_month = (24 * 60 * 60 * 30)
+  secs_in_year = (24  * 60 * 60 * 365)
+  secs_in_day = (24 * 60 * 60)
+  cid_size = 32 + 1 + 1 + 1
+  to_i = 1024/1000
```

Comment:
I think this should actually be `(1024 * 1024 * 1024) / (1000 * 1000 * 1000)`.

Reply:
I see. This depends on whether we multiply gbs mbs and kbs